tests/xcm-transfer/shared.ts

```diff
@@ -19,6 +18,27 @@ type TestType =
   | PolkadotParaTestType
   | PlaygroundTestType
 
+// TODO: figure out whey using chopsticks-testing impl will not find 'XcmVersionedXcm'
```

Comment:
Is this for future tasks?

Reply:
don't know. for some reason current runtime doesn't work. Tried upcoming runtime and it works. So this is tmp approach

Reply:
ok, let me create an issue for it. could be resolved later